### ISSUE TITLE
Unable to start QuadIt

### ISSUE THREAD

**Issue by escoand** [ubiquitous-factory/quadit#29]
Hi,

I tried to switch from FetchIt to QuadIt because it sounds really promising.
But had to fight with multiple problems. 

```
$ curl -o ~/.quadit https://raw.githubusercontent.com/ubiquitous-factory/quadit/main/samples/config.yaml
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: Failed to create the file /home/admin/.quadit: Is a directory
100   277  100   277    0     0    914      0 --:--:-- --:--:-- --:--:--   914
curl: (23) Failure writing output to destination
```
Is this working with your version of `curl`? I just appended the filename.

Then I wasn't able to pull the quadit image from quay.io. Is it maybe just private? So I had to build it on my own.

And at the end the systemd service crashed after no time. I searched for the command and started by hand:
```
$ podman run --name=quadit --cidfile=/run/user/1000/quadit.cid --replace --rm --cgroups=split --sdnotify=conmon --security-opt label:disable --userns keep-id -v quadit-volume:/opt -v /home/admin/.quadit:/opt/config -v /home/admin/.config/containers/systemd:/opt/containers -v /run/user/1000/podman/podman.sock:/run/podman/podman.sock --label io.containers.autoupdate=registry --env HOME=/home/admin --env JOB_PATH=/tmp --env PODMAN_SYSTEMD_UNIT=quadit.service --env XDG_RUNTIME_DIR=/run/user/1000 --mount type=tmpfs,destination=/run/user/1000,rw=true --mount type=bind,source=/sys/fs/cgroup,destination=/sys/fs/cgroup,ro=true --mount type=bind,source=/run/user/1000/systemd,destination=/run/user/1000/systemd,rw=true --init-path /usr/sbin/init --pid host --privileged quay.io/ubiquitous-factory/quadit:v0.1.3
quadit Server Edition v0.1.4
This process accepts no arguments.
See documentation https://github.com/ubiquitous-factory/quadit/
```

Could you help me at this point. Currently I'm a bit limited in time to investigate this further.

Thanks in advance.

**Comment by No9** [comment]
Hi 
Thanks for your words of support - I had some other work over the past few weeks but I'm going to be working on quadit for the rest of the year so can collaborate on this. 

`curl -o ~/.quadit https://raw.githubusercontent.com/ubiquitous-factory/quadit/main/samples/config.yaml`

Should be 

`$ curl -o ~/.quadit/config.yaml https://raw.githubusercontent.com/ubiquitous-factory/quadit/main/samples/config.yaml`

The original command was trying to write a file to the directory location which is not correct. 

I had to pull the images as there was an issue with licensing of downstream components that weren't relevant to this project.

I'll clean up the images and reinstate them soon and come back to this.

**Comment by No9** [comment]
OK I've updated the image definition and the container registry location and fixed a bug and cut a new release. 
https://github.com/ubiquitous-factory/quadit/releases/tag/v0.1.5 

The install instructions https://github.com/ubiquitous-factory/quadit?tab=readme-ov-file#install should work now. 
The quadit image is now available from quay and is public 
https://quay.io/repository/mehal_tech/quadit?tab=tags 
I've moved the repository location to my companies org as I will be using quadit as part of a wider solution but the idea is to keep it as standalone as possible.

Just to restate - there will be breaking changes coming into this as I figure out canary deploys and I broaden out the test coverage but I hope it's useful and I'd really appreciate any feedback. Good or Bad :)  

I've revalidated this on Fedora 40 but haven't had a chance to test on ubuntu yet.
